backend/test/helpers.ts

```diff
@@ -46,3 +47,15 @@ export const resetEntity = async (entity: any) => {
 export const resetToken = () => {
   context.token = ''
 }
+
+export const setClientRequestTime = (tm: Date) => {
```

Comment:
in the context a string is written as it will be done by the client.
the signature with a date type for invokations from the test classes helps to use a flexible date without always converting it to a string before, because it will be done implicit